ioBroker.discovery.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does this file really required?

BTW: Very good prepared PR :)

Reply:
Nope ... I forget to add the .iml (Intellij IDE) to .gitignore